test/travis.sh

```diff
@@ -40,53 +36,25 @@ fi
 # At this point, if anything fails, the test should fail
 set -e
 
-if ! $ONLY_RULESETS_CHANGED; then
-  echo >&2 "Core code changes have been made."
-
-  if [ "$TEST" == "eslint" ]; then
-    echo >&2 "Running ESLint on the Chromium extension."
-    docker run --rm -ti -v $(pwd):/opt node bash -c "cd /opt/utils/eslint && npm install && ./node_modules/.bin/eslint ../../chromium"
-  fi
-
-  if [ "$TEST" == "firefox" ]; then
-    echo >&2 "Running firefox test suite."
-    docker_build
-    docker run --rm -ti -v $(pwd):/opt -e FIREFOX=/$FIREFOX/firefox/firefox httpse bash -c "test/firefox.sh"
-  fi
-
-  if [ "$TEST" == "chromium" ]; then
-    echo >&2 "Running chromium test suite."
-    docker_build
-    # --privileged is required here because chromium requires kernel lxc access
-    docker run --rm -ti -v $(pwd):/opt --privileged httpse bash -c "test/chromium.sh"
-  fi
+if [ "$TEST" == "rules" ]; then
+  echo >&2 "Performing comprehensive coverage test."
+  docker run --rm -ti -v $(pwd):/opt httpse python utils/ruleset_filenames_validate.py
+  docker run --rm -ti -v $(pwd):/opt httpse bash -c "utils/validate.sh"
+  docker run --rm -ti -v $(pwd):/opt httpse bash -c "test/rules.sh"
+  docker run --rm -ti -v $(pwd):/opt node bash -c "cd /opt && node utils/normalize-securecookie.js"
+  [ `git diff --name-only | wc -l` -eq 0 ]
 fi
-# Only run test if something has changed.
-if [ "$RULESETS_CHANGED" ]; then
```

Comment:
We always run the tests now.

This avoids a possible issue where the ruleset validation code changes, but the rulesets don't. So the new validation code would not get tested against existing rulesets.